docs/Integrations/Authentication/authentication.md

```diff
@@ -0,0 +1,29 @@
+
+
+# Authentication 
+Authentication allows users to create an account and log in to your app. We support Firebase, Supabase, and Custom Authentication. Under App Settings, choose Authentication, and then you can enable authentication. Choose the initial pages for the entry and logged-in states, and complete further steps if applicable.
```

Comment:
I dont think we should have pages tied to section headers, for consistency. So move this into an "authentication overview" page

Reply:
Done!